index.html

```diff
@@ -200,7 +205,7 @@ <h3 class="article__title article__title--sidebar">
       <!-- end of row -->
     </div>
 
-    <div class="footer container">
+    <footer class="footer container">
```

Comment:
Now you've correctly applied semantic HTML ou could fix your styling so as to apply it directly to the `footer` tag. This would mean you'd delete `footer` from the class attribute above and then change the CSS blocks in your CSS file from `.footer` to `footer`. However either version of the code works.